ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -317,30 +317,201 @@
     },
     {
       id: 'E8S Refulgent Chain',
-      regex: Regexes.gainsEffect({ effect: 'Refulgent Chain' }),
-      regexDe: Regexes.gainsEffect({ effect: 'Lichtfessel' }),
-      regexFr: Regexes.gainsEffect({ effect: 'Chaînes de Lumière' }),
-      regexJa: Regexes.gainsEffect({ effect: '光の鎖' }),
-      condition: Conditions.targetIsYou(),
-      suppressSeconds: 1,
-      infoText: {
-        en: 'Chain on YOU',
-        de: 'Kette auf DIR',
-        fr: 'Chaine sur VOUS',
-        ko: '사슬 대상자',
-        cn: '连线',
+      regex: Regexes.tether({ id: '006E' }),
+      condition: function(data, matches) {
+        if (data.options.cactbote8sLightRampantStrat === undefined || data.options.cactbote8sLightRampantStrat === 'none')
+          return data.me == matches.source;
+
+        if (data.options.cactbote8sLightRampantStrat === 'sharingan') {
```

Comment:
I'm not up to speed on the various chain and orb strats going around. For chain, this could be referred to as "hourglass", but I'm not sure what to call the orb strat then. And given that, does it make sense as well to have a separate setting for the conals/earthshakers/whatever we're calling them these days?